src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,193 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Wrapper for the Tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+    private Storage storage;
+    private Ui ui;
+
+    public TaskList(Storage storage, Ui ui) {
+        this.tasks = new ArrayList<>(100);
+        this.storage = storage;
+        this.ui = ui;
+
+        this.storage.initialiseTasks(tasks);
+    }
+
+    public int sizeOf() {
+        return tasks.size();
+    }
+
+    /**
+     * Displays contents of list to screen.
+     */
+    public void displayList() {
+        String line = "____________________________________________________________";
+        String indent = "    ";
+
+        if (tasks.size() == 0) {
+            ui.toScreen("You currently have no tasks.");
+            return;
+        }
+
+        System.out.println(indent + line + "\n" +
+                indent + "Here are the tasks in your list: ");
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println(indent + indent + (i + 1) + "." + tasks.get(i).toString());
+        }
+
+        System.out.println(indent + line);
+    }
+
+    /**
+     * Marks task at given index as done.
+     *
+     * @param index Index of task.
+     */
+    public void doneTask(int index) {
```

Comment:
Might be good to give this a better name
```suggestion
    public void markTaskAsDone(int index) {
```